blockchain/chain_sync/src/network_context.rs

```diff
@@ -70,6 +70,26 @@ impl SyncNetworkContext {
             .map(|fts| fts.tipset().map_err(|e| e.to_string()))
             .collect::<Result<_, _>>()
     }
+    /// Send a blocksync request for full tipsets (includes messages)
+    pub async fn blocksync_fts(
+        &mut self,
+        peer_id: PeerId,
+        tsk: &TipSetKeys,
+    ) -> Result<FullTipset, String> {
+        let bs_res = self
+            .blocksync_request(
+                peer_id,
+                BlockSyncRequest {
+                    start: tsk.cids().to_vec(),
+                    request_len: 1,
+                    options: 3,
```

Comment:
not needed to change now, but for this and the one above, options should be constants and options set based on those, example:
```
BLOCKS = 1
MESSAGES = 2
..
options: BLOCKS | MESSAGES
```
something like that for readability/ future proofing, can do in this PR if you want, or I can make the change later